_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -2367,9 +2367,7 @@
 /obj/item/seeds/cotton{
 	yield = 5
 	},
-/obj/item/seeds/cotton{
-	yield = 5
-	},
+/obj/item/seeds/cotton/durathread,
```

Comment:
Please make their yield be 5 so they dont fucked over by Rng

Reply:
No.
First, durathread can also be used to make armors, which is very nice but I'm not planning to make ashwalkers easier by giving them higher yield durathead.
Second, that sort of RNG usually happens when the plant is either getting old or going without nutriment/water for an extended period of time. And that's your loss.